README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
```

Comment:
@evocateur what if both npmClientArgs and `--` are provided, should we append one to another or replace? I vote for append as it'll work like "common arguments you always want to pass".